addons/hr_recruitment_survey/data/survey_demo.xml

```diff
@@ -2,7 +2,7 @@
 <odoo><data noupdate="1">
     <record id="survey_recruitment_form" model="survey.survey">
         <field name="title">Recruitment Form</field>
-        <field name="user_id" ref="base.user_admin"/>
+        <field name="user_ids" eval="[(4, ref('base.user_admin'))]"/>
```

Comment:
Shouldn't we keep it void so that demo user has access to it ? For demo seems it is better to have some demo data available.